README.md

```diff
@@ -3,7 +3,7 @@ Workshop RBFA
 
 Introduction
 ------------
-This repository was made during the onboarding at the Royal Belgian Football Association. The main goal of tis workshop is to learn how to use some basic functionalities of teh Google Cloud Platform like Bigquery, Storage and Datastore.
+This repository is an onboarding project at the Royal Belgian Football Association. The main goal of tis workshop is to learn how to use some basic functionalities of teh Google Cloud Platform like Bigquery, Storage and Datastore.
```

Comment:
'The repository is an onboarding project for developers joining the RBFA dev team'

Reply:
The project aims to provide a new member with the necessary tools to join the RBFA team